website/docs/d/vpc_ids.html.markdown

```diff
@@ -0,0 +1,57 @@
+---
+layout: "aws"
+page_title: "AWS: aws_vpc_ids"
+sidebar_current: "docs-aws-datasource-vpc-ids"
+description: |-
+    Provides a list of VPC Ids in a region
+---
+
+# Data Source: aws_vpc_ids
+
+`aws_vpc_ids` provides a list of vpc ids
```

Comment:
Nitpick: This sentence seems redundant with the one below. Also, the casing of VPC IDs is different between various sentences in the documentation page.

Reply:
done